public/main.js

```diff
@@ -1,31 +1,45 @@
 $(document).on('ready', function(){
 	submitAnswer();
+    createNewMessage();
```

Comment:
definitely not `create` new message. Because you are not creating it here, you are receiving it.